txs/cache.go

```diff
@@ -727,13 +732,16 @@ func (c *cache) ApplyLayer(
 		}
 		acctResetDuration.Observe(float64(time.Since(t1)))
 	}
-	for principal := range skippedByPrincipal {
-		if _, ok := byPrincipal[principal]; ok {
+
+	for principal, cash := range c.pending {
```

Comment:
what is cash?

Reply:
don't want to call it cache (clash with the struct name) so use it as it sounds the same. anyhow, renamed to `accCache`